person.js

```diff
@@ -80,6 +86,40 @@ var Person = new Lang.Class({
         }));
     },
 
+    _getGitLabInfo: function() {
+        let _httpSession = new Soup.Session({user_agent: 'jwendell/gnome-shell-extension-timezone'});
+        let url = 'https://gitlab.com/api/v4/avatar?email=%s'.format(this.gitlab);
+        let message = new Soup.Message({method: 'GET', uri: new Soup.URI(url)});
+        if (this._gitlabToken) {
+            message.request_headers.append("Authorization", "token " + this._gitlabToken);
```

Comment:
According to https://docs.gitlab.com/ee/api/#oauth2-tokens should this header be `Authorization: Bearer <your_access_token>` instead?